packages/login/theme/material/vaadin-login-overlay-styles.js

```diff
@@ -337,9 +337,9 @@ const loginFormWrapper = css`
     }
 
     :host([theme~='with-overlay']) [part='form'] ::slotted(form) {
-      justify-content: space-around;
-      margin-top: 20px;
-      min-height: 250px;
+      justify-content: space-evenly;
+      margin-top: 15px;
+      min-height: 150px;
```

Comment:
Previously, the submit button was also part of the form. Decreased min-height and adjusted margin to compensate. It's hard to achieve 100% pixel perfect match because of flexbox usage.

Reply:
UPD: discussed with @vursen that we don't need `min-height` after all, so I removed it.